docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -1,29 +1,111 @@
-# Grid Layout
+# Grid layout
+
+> [!NOTE]
+> Only `Columns` property is supported on Tizen 5.5.
 
 `GridLayout` is a grid box for the two dimensional layout. It constraints the x and y position, width, and height of the child actors.
```

Comment:
This line is repeated twice, please fix it.

Reply:
Fixed.